support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -338,7 +338,7 @@ function getAmountsTestVariant(
 		// Check if we actually want to track this test
 		const pathMatches = targetPageMatches(
 			path,
-			'/??/contribute|thankyou(/.*)?$',
```

Comment:
@tomrf1 fyi this is the change we discussed yesterday

Reply:
This will mean we now start tracking participation in multi-variant amounts tests on the generic checkout & one-time-checkout in Ophan page views.